src/VscProcess.cpp

```diff
@@ -89,7 +89,15 @@ VscProcess::VscProcess() : myEStopState(0)
 
   // Publish Emergency Stop Status
   estopPub = rosNode.advertise<std_msgs::UInt32>("safety/emergency_stop", 10);
-
+  
+  // Publish Remote Status
+  vscModePub = rosNode.advertise<std_msgs::UInt32>("safety/vsc/mode", 10);
+  batteryLevelPub = rosNode.advertise<std_msgs::UInt32>("safety/src/battery/level", 10);
+  batteryChargingPub = rosNode.advertise<std_msgs::Bool>("safety/src/battery/is_charging", 10);
+  vscConnectionStrengthPub = rosNode.advertise<std_msgs::UInt32>("safety/vsc/connection_strength", 10);
+  srcConnectionStrengthPub = rosNode.advertise<std_msgs::UInt32>("safety/src/connection_strength", 10);
```

Comment:
Can we condense this into a single ROS message?

Reply:
I was hoping you'd say that.
Would you rather an array or a custom msg?

Reply:
It'd simplify the subscriber in the system status monitor if we condense it.

Reply:
I'd prefer a single custom message. The issue with arrays is that you need to keep track of what field is in what index and how to interpret it, so they end up causing headaches in areas where you don't really NEED the abstraction.

Reply:
Also, you can get rid of the struct if you use the ROS message

Reply:
Great! I wanted to do that but didn't know if it would be overkill.
`greenzie_msgs` seems like a good place for it but let me know if you'd rather it in the hri fork.

Reply:
It needs to be here in the HRI fork, otherwise you'll end up with a circular dependency, which is NOT good.